packages/redis-v5/commands/cli.js

```diff
@@ -128,13 +128,24 @@ function maybeTunnel (redis, config) {
   let bastions = match(config, /_BASTIONS/)
   let hobby = redis.plan.indexOf('hobby') === 0
   let uri = url.parse(redis.resource_url)
+  let version = redis.version
 
   if (bastions != null) {
     return bastionConnect({ uri, bastions, config })
   } else {
     let client
-    if (!hobby) {
-      client = tls.connect({ port: parseInt(uri.port, 10) + 1, host: uri.hostname, rejectUnauthorized: false })
+    if (version >= '6.0') {
+      client = tls.connect({
```

Comment:
do redis 6 `redis.resource_url`s indicate ssl in the scheme (e.g. `https://`, `ssl://`, `tls://`, etc)? we could pull it off of `uri.scheme` instead of relying on the version check if that's the case

Reply:
the api also responds with `prefers_native_tls` so we can use that instead of the version check

Reply:
the version check is actually wrong as coded since a hobby-dev on version 6 doesn't support tls

Reply:
@coreypurcell thanks!

@fivetanley I like that idea (switch on `if (prefer_native_tls)`) since that is what we are trying to do here. For now, this flag will only be present on Redis 6. What do you think?